### PR TITLE
User staging table

### PR DESCRIPTION
**Pull request by benrudolph**
@gcapalbo needed for the initial draft of the app status fact table (requires the user dim)

cc: @proteusvacuum

### CODE REVIEW

File: corehq/warehouse/models/staging.py [hunk 1]
```diff
@@ -74,6 +76,49 @@ def raw_record_iter(cls, start_datetime, end_datetime):
         return iter_docs(Group.get_db(), group_ids)
 
 
+class UserStagingTable(StagingTable):
+    slug = 'user_staging'
+
```
Comment (gcapalbo) [inline corehq/warehouse/models/staging.py]: i think it would be good to bring in base_doc on the staging table, and transform to a deleted flag on the dim

**Comment by gcapalbo** [comment]
:+1: just one comment otherwise looks good

File: corehq/warehouse/models/staging.py (hunk 1 above)
Comment (benrudolph) [inline corehq/warehouse/models/staging.py]: 👍 good call

**Review by gcapalbo — APPROVED** [review]

**Comment by gcapalbo** [comment]
(build failure looks related)

**Comment by benrudolph** [comment]
merging because this was approved and last commit was just test cleanup code

**Pull request merged**